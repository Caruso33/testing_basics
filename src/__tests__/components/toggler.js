import Toggler from "../../components/toggler.jsx";
import { render } from "@testing-library/react";

describe("Toggler Func component", () => {
  test("should render with default props", () => {
    const component = <Toggler onToggle={jest.fn()}>Child</Toggler>;

    const wrapper = render(component);
    const { baseElement } = wrapper;
    expect(baseElement).toMatchSnapshot();
  });
});
