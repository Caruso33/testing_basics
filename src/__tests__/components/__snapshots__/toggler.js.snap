// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggler Func component should render with default props 1`] = `
<body>
  <div>
    <button
      class="sc-bdVaJa iTSWAp"
      data-test="button"
      on="0"
    >
      <span
        class="sc-bwzfXH kabNnl"
      >
        Toggle Me
      </span>
    </button>
  </div>
</body>
`;
